corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -121,5 +121,18 @@ <h1 class="page-title"><%- title %></h1>
 <script type="text/template" id="breadcrumb-list-template">
   <ol class="breadcrumb">
     <li class="js-home breadcrumb-text"><i class="fa fa-home"></i></li>
+    <div class="pull-right dropdown dropdown-menu-right" id="form-menu">
+    </div>
   </ol>
 </script>
+
+<script type="text/template" id="form-menu-template">
+   <i class="glyphicon glyphicon-option-vertical dropdown-toggle" data-toggle="dropdown"></i>
```

Comment:
Glyphicons are built into bootstrap, but HQ uses a different icon library, [fontawesome](https://fontawesome.com/v4.7.0/cheatsheet/). I don't see a three-dot icon, `fa-bars` is probably the closest. Something like `fa-chevron-down` or `fa-caret-down` would work, too.